render-configs.sh

```diff
@@ -1,32 +1,136 @@
+# ./render-configs.sh -a integration -r tools -i
+
+# If you want a set up locally, you can use the following settings:
+# ./render-configs.sh -a dev -r dev
+# Then, refresh your z-shell configuration (`source ~./zshrc`)
+# ./gradlew bootRun
+
+AZURE_ENV=dev
+RBS_ENV=tools
+COPY_INTELLIJ_ENV_VARS=n
+
+while getopts ":a:r:i" option; do
+  case $option in
+    a)
+      AZURE_ENV=$OPTARG
+      ;;
+    r)
+      RBS_ENV=$OPTARG
+      ;;
+    i)
+      COPY_INTELLIJ_ENV_VARS=y
+      ;;
+    *)
+      echo "Usage: $0 [-a (dev|integration)] [-r (tools|dev)] [-i]"
+      exit 1
+      ;;
+  esac
+done
+
+# ========================
+# Azure Credentials
+# ========================
+# If you want to run Azure Integration tests locally, you need to point to the "integration" environment
+# Options: 'dev', 'integration'
+
+if [[ "${AZURE_ENV}" == "dev" ]]; then
+    AZURE_SYNAPSE_WORKSPACENAME=tdr-synapse-east-us-ondemand.sql.azuresynapse.net
+elif [[ "${AZURE_ENV}" == "integration" ]]; then
+    AZURE_SYNAPSE_WORKSPACENAME=tdr-snps-int-east-us-ondemand.sql.azuresynapse.net
+else
+    echo "Invalid Azure environment: $AZURE_ENV"
+    exit 1
+fi
+# writing this values to a tmp file so the value can match the set azure environment
+echo $AZURE_SYNAPSE_WORKSPACENAME > "/tmp/azure-synapse-workspacename.txt"
+
+
+vault read -field=tenant-id secret/dsde/datarepo/"$AZURE_ENV"/azure-application-secrets \
+    > "/tmp/jade-dev-tenant-id.key"
+AZURE_CREDENTIALS_HOMETENANTID=$(cat "/tmp/jade-dev-tenant-id.key")
+
+
+vault read -field=client-id secret/dsde/datarepo/"$AZURE_ENV"/azure-application-secrets \
+    > "/tmp/jade-dev-client-id.key"
+AZURE_CREDENTIALS_APPLICATIONID=$(cat "/tmp/jade-dev-client-id.key")
+
+
+vault read -field=client-secret secret/dsde/datarepo/"$AZURE_ENV"/azure-application-secrets \
+    > "/tmp/jade-dev-azure.key"
+AZURE_CREDENTIALS_SECRET=$(cat "/tmp/jade-dev-azure.key")
+
+
+vault read -field=synapse-sql-admin-user secret/dsde/datarepo/"$AZURE_ENV"/azure-application-secrets \
+    > "/tmp/jade-dev-synapse-admin-user.key"
+AZURE_SYNAPSE_SQLADMINUSER=$(cat "/tmp/jade-dev-synapse-admin-user.key")
+
+
+vault read -field=synapse-sql-admin-password secret/dsde/datarepo/"$AZURE_ENV"/azure-application-secrets \
+    > "/tmp/jade-dev-synapse-admin-password.key"
+AZURE_SYNAPSE_SQLADMINPASSWORD=$(cat "/tmp/jade-dev-synapse-admin-password.key")
+
+
+vault read -field=synapse-encryption-key secret/dsde/datarepo/"$AZURE_ENV"/azure-application-secrets \
+    > "/tmp/jade-dev-synapse-encryption-key.key"
+AZURE_SYNAPSE_ENCRYPTIONKEY=$(cat "/tmp/jade-dev-synapse-encryption-key.key")
+
+# ========================
+# Google Credentials
+# ========================
 vault read -format=json secret/dsde/datarepo/dev/sa-key.json \
     | jq .data | tee /tmp/jade-dev-account.json \
     | jq -r .private_key > /tmp/jade-dev-account.pem
```

Comment:
You could avoid a call to `jq` here
```suggestion
vault read -field data -format=json secret/dsde/datarepo/dev/sa-key.json \
    | tee /tmp/jade-dev-account.json \
    | jq -r .private_key > /tmp/jade-dev-account.pem
```